test/library/packages/ConcurrentMap/testGetRemove.chpl

```diff
@@ -0,0 +1,16 @@
+use ConcurrentMap;
+
+class C {
+  var i: int;
+}
+
+var m = new shared ConcurrentMap(string, shared C);
+var a = new C(1);
```

Comment:
This test is failing because `a` and `b` aren't shared. It turns out the map also has problems in `getAndRemove` because it tries to default-initialize non-nilable classes. Let's avoid this issue for now by making this a `ConcurrentMap(string, int)` instead.